mr-plonky2-circuits/src/query2/revelation/public_inputs.rs

```diff
@@ -146,6 +152,9 @@ impl<'a, const L: usize> RevelationPublicInputs<'a, Target, L> {
             b.register_public_input(nft_id.0);
         }
         b.register_public_inputs(&lpn_latest_block.to_targets().arr);
+        // Register the 16 padded items (2 * uint256).
+        let zero = b.zero();
+        b.register_public_inputs(&[zero; 2 * PACKED_U256_LEN]);
```

Comment:
I think that it would be more appropriate to use `2*u256::NUM_LIMBS` now that we have integrated the unit256 gadget

Reply:
Fixed in commit [e120525](https://github.com/Lagrange-Labs/Euclid-database/pull/13/commits/e120525ee1e20b8dbfc165a1406e36992c39dfbd).